<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #f0f0f0;
    border: 1px solid #aaa;
    border-radius: 5px;
    padding: 6px 12px;
}

QTextEdit {
    background-color: #fff;
    border: 1px solid #ccc;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Liczba miejsc:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stawka za  godzinę (zł):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rozpocznij</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLiczbaMiejsc">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStawka">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>135</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #f0f0f0;
    border: 1px solid #aaa;
    border-radius: 5px;
    padding: 6px 12px;
}

QTextEdit {
    background-color: #fff;
    border: 1px solid #ccc;
}
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
